packages/kbn-proc-runner/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@kbn/proc-runner",
+  "main": "./index.js",
+  "version": "1.0.0",
+  "license": "Apache-2.0",
+  "private": true,
```

Comment:
Since this package is written with esm we need at least a minor build step to create node 8 compatible js out of the source.

I recommend checking out to how [`@kbn/datemath`](https://github.com/elastic/kibana/blob/master/packages/kbn-datemath/package.json#L9-L10) does it. We'll need `build` and `kbn:bootstrap` scripts that build into `target`. Unlike `@kbn/datemath` we should just use the `@kbn/babel-preset/node` rather than pulling in `babel-preset-env` directly.

Reply:
great, thanks for the info. ++